ui-manchette/src/components/Manchette.tsx

```diff
@@ -8,7 +8,7 @@ import WaypointList from './WaypointList';
 import type { InteractiveWaypoint, WaypointMenuData } from '../types';
 
 type ManchetteProps = {
-  waypoints: InteractiveWaypoint[];
+  contents: (InteractiveWaypoint | React.ReactNode)[];
```

Comment:
Hum I think that in the specs, we ask for this type (an array composed of either an array of interactive waypoints or a react node)
```suggestion
  contents: (InteractiveWaypoint[] | React.ReactNode)[];
```

Reply:
Ok hum actually, I think it's better to keep it like this, sorry ! It will be easier to use for the users which don't have custom jsx components. :+1: